compiler/front_end/module_ir_test.py

```diff
@@ -4057,12 +3984,12 @@ def _check_source_location(source_location, path, min_start, max_end):
     for name, field in (("start", start), ("end", end)):
         if not field:
```

Comment:
similar comment as above, should this be `field is None`

Reply:
The case where `not field and field is not None` is covered a few lines above, in the `if not source_location.start`/`.end`.

Reply:
Oh right I see, we have an overall "end is missing"